pip/basecommand.py

```diff
@@ -109,7 +109,12 @@ def main(self, args):
         options, args = self.parse_args(args)
 
         if options.quiet:
-            level = "WARNING"
+            if options.quiet == 1:
+                level = "WARNING"
+            if options.quiet == 2:
```

Comment:
@xavfernandez Yeah, looking back, I'm not sure why I didn't use an `elif`. It obviously makes sense to use it. Should I submit a separate pull request, add it to this one, or just let it be?